python/pyspark/ml/tuning.py

```diff
@@ -207,6 +210,205 @@ def _to_java_impl(self):
         return java_estimator, java_epms, java_evaluator
 
 
+class _ValidatorSharedReadWrite:
+
+    @staticmethod
+    def saveImpl(path, instance, sc, extraMetadata=None):
+        from pyspark.ml.classification import OneVsRest
+        numParamsNotJson = 0
+        jsonEstimatorParamMaps = []
+        for paramMap in instance.getEstimatorParamMaps():
+            jsonParamMap = []
```

Comment:
L219 and L221 are called maps but are actually lists. Can you fix the names?

Reply:
they're maps in "json" style (can be directly serialized by json.dumps()) . So named like jsonXXX. It is similar to other existing codes